src/components/weather/DailyWeather.tsx

```diff
@@ -35,7 +36,10 @@ export const DailyWeather = ({ date, icon, temperatures }: Props) => {
           <WrapperRow center>
             <View style={styles.iconContainer}>
               <Image
-                source={{ uri: `https://openweathermap.org/img/wn/${icon}@2x.png` }}
+                source={{
+                  uri: `https://openweathermap.org/img/wn/${icon}@2x.png`,
+                  captionText: desctription
```

Comment:
please share an example of data coming from the api.

Reply:
Here are some examples of descriptions from the API:
```
Leichter Regen
Bedeckt
Mäßig bewölkt
Klarer Himmel
Ein paar Wolken
Überwiegend bewölkt